pr_insight/git_providers/utils.py

```
@@ -49,10 +58,40 @@
+                    body += f"\n\n**Configuration content:**\n\n```toml\n{configuration_file_content}\n```\n\n"
+                get_logger().warning(f"Sending a 'configuration error' comment to the PR", artifact={'body': body})
+                # git_provider.publish_comment(body)
+                if hasattr(git_provider, 'publish_persistent_comment'):
+                    git_provider.publish_persistent_comment(body,
+                                                            initial_header=header,
+                                                            update_header=False,
+                                                            final_update_message=False)
+                else:
+                    git_provider.publish_comment(body)
+    except Exception as e:
+        get_logger().exception(f"Failed to handle configurations errors", e)
```

Ruff

f-string without any placeholders

(F541)